database/Seed.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const faker = require('faker');
+
+
+const itemModel = require('./mongooseModel.js');
+
+mongoose.connect('mongodb://localhost/items');
+
+for (let i = 0; i < 100; i += 1) {
+  const obj = {
+    id: i,
+    name: faker.commerce.productName(),
+    photo: `https://fec-project-photos.s3.us-east-2.amazonaws.com/image${i}.jpg`,
+  };
+  itemModel.item.create(obj);
```

Comment:
The way you've set up the seeding, each document will contain a single photo and name. How do you plan to have multiple (x - number of) images render within your container? Making multiple /get requests? I suggest you refactor the seeding script and implement a function to create random group sets of id/image/photo objects.

Reply:
I had planned to use State within my Client side react app hold the items as they are requested (ie viewing another new item). The items on State can pass the photos to the various components and render them as needed. Since the widget is fairly simple I don't need All of the images for each Item. I only need the "main" image for each item that was recently viewed. The one photo stored on my item represents the "main" image. By "main" image I mean the first image that you see when an item is viewed.